diesel_compile_tests/tests/compile-fail/returning_clause_requires_selectable_expression.rs

```diff
@@ -0,0 +1,44 @@
+#[macro_use]
+extern crate diesel;
+
+use diesel::*;
+use diesel::pg::PgConnection;
+
+table! {
+    users {
+        id -> Integer,
+        name -> VarChar,
+    }
+}
+
+pub struct NewUser(String);
+
+Insertable! {
+    (users)
+    pub struct NewUser(#[column_name(name)] String,);
+}
+
+table! {
+    non_users {
+        id -> Integer,
+        noname -> VarChar,
+    }
+}
+
+fn main() {
+    let connection = PgConnection::establish("").unwrap();
+
+    delete(users::table.filter(users::columns::name.eq("Bill")))
+        .returning(non_users::columns::noname);
+    //~^ ERROR: SelectableExpression
```

Comment:
I'm curious if that means that there happens to be a `: ` in the right spot or if the test isn't running. XD